doc/getting_started/installation_win.rst

```diff
@@ -73,7 +73,7 @@ environment for Windows. Follow the steps below to set it up:
       $ curl -O 'https://bootstrap.pypa.io/get-pip.py'
       $ ./get-pip.py
       $ rm get-pip.py
-      $ pip install --user -r scripts/requirements.txt
+      $ pip install --user -r zephyr/scripts/requirements.txt
```

Comment:
No. We make no assumption of the root directory name where the Zephyr source is cloned.

Reply:
At least it is not obvious that the `scripts` directory is owned by `zephyr`. One can be confused by the current command. Maybe it should be stated that the directory is actually the installation directory of the Zephyr project. `YOUR_ZEPHYR_PROJ_INSTALL_PATH/scripts/requirements.txt`

Reply:
add `$(ZEPHYR_BASE)/scripts` instead of plain 'zephyr'

Reply:
...as long as you've sourced the zephyr-env.sh script :)
It also indicates there's a Zephyr base directory even if you haven't sourced the script yet, so I'd go with Anas' suggestion for the edit.